homeassistant/components/google_assistant_sdk/__init__.py

```diff
@@ -164,7 +164,7 @@ async def async_process(
             await session.async_ensure_token_valid()
```

Comment:
Why do we still have a session if we're not going to use that when using the JSON file for credentials? Why should the user set up the integration via a web auth flow when the credentials from that flow won't be used in the integration.

Reply:
Fixed. Reauth can't work if you use the creds json file.